src/api/app/controllers/webui/users_controller.rb

```diff
@@ -140,6 +140,20 @@ def change_password
     end
   end
 
+  def rss_secret
+    user = User.session!
+
+    verb_prefix = user.rss_secret.present? ? 're-' : ''
+
+    respond_to do |format|
+      if user.regenerate_rss_secret
```

Comment:
`User.regenerate_rss_secret` will raise an exception, instead of returning `false`...

Reply:
removed